http/http_connection.hpp

```diff
@@ -278,6 +274,12 @@ class Connection :
 
     void start()
     {
+        if (connectionCount >= 100)
+        {
+            BMCWEB_LOG_CRITICAL << this << "Max connection count exceeded.";
```

Comment:
Ahh, we get the count limit message here so we're good on that one.

Reply:
Yeah.